scripts/release.py

```diff
@@ -0,0 +1,43 @@
+import os
+from pathlib import Path
+from subprocess import getoutput
+
+
+def make_package(name, verbose=False):
+    orig_dir = os.getcwd()
+    base_dir = os.path.join(os.path.abspath(os.path.dirname(Path(__file__))), "..")
+    os.chdir(base_dir)
+
+    with open(".VERSION", "r") as f:
+        __version__ = f.readline()
+
+    output = getoutput(f"python3 {name}/setup.py sdist")
```

Comment:
I can't really image this would ever be a problem but, to just make sure we're using the same version of python, it might be safer to do:
```
output = getoutput(f"{sys.executable} {name}/setup.py sdist")
```

Reply:
👍